client/src/nv_ingest_client/primitives/tasks/embed.py

```diff
@@ -58,6 +60,17 @@ def __init__(self, text: bool = None, tables: bool = None, filter_errors: bool =
                 "'tables' parameter is deprecated and will be ignored. Future versions will remove this argument."
             )
 
+        embedding_nim_endpoint = os.getenv("EMBEDDING_NIM_ENDPOINT", "https://integrate.api.nvidia.com/v1")
+        if embedding_nim_endpoint == "":
+            self._embedding_nim_endpoint = "https://integrate.api.nvidia.com/v1"
+        elif embedding_nim_endpoint[:7] != "http://" and embedding_nim_endpoint[:8] != "https://":
```

Comment:
would prefer to see python str `.startswith('http://')` and `.startswith('https://')'` used here